kuma/users/checks.py

```diff
@@ -0,0 +1,35 @@
+import stripe
+from django.conf import settings
+from django.core.checks import Error, register
+
+
+STRIPE_ERROR_ID = "kuma.users.E001"
```

Comment:
nit: It doesn't need the `_ID` suffix, because they're all IDs. I think just `STRIPE_ERROR` is good enough. If we had a bunch of other things that could go wrong, it ought to be called something more description like `STRIPE_PLAN_UNEXPECTED_ERROR`.